Sources/Components/Components/Watchlist/Filter/WKWatchlistFilterView.swift

```diff
@@ -22,7 +22,8 @@ struct WKWatchlistFilterView: View {
                             doneAction()
                         }) {
                             Text(viewModel.localizedStrings.doneTitle)
-                                .foregroundColor(Color(theme.text))
+                                .fontWeight(Font.Weight.semibold)
```

Comment:
This should probably flow through `WKFont`.

Reply:
Agh, good catch. I pushed a fix.